Backend/app/rider-platform/rider-app/Main/src/Storage/Queries/Quote.hs

```diff
@@ -18,13 +18,15 @@ module Storage.Queries.Quote where
 import Data.Tuple.Extra
 import Domain.Types.DriverOffer
 import Domain.Types.Estimate
+import Domain.Types.FarePolicy.FareProductType
 import Domain.Types.Quote
 import Domain.Types.SearchRequest
 import Kernel.Prelude
 import Kernel.Storage.Esqueleto as Esq
 import Kernel.Types.Id
+import Storage.Queries.DriverOffer
```

Comment:
Query modules should not import each other (except FullEntityBuilders), define this func here locally.